File: src/main/scala/com/github/mauricio/postgresql/util/Log.scala
package com.github.mauricio.postgresql.util

import org.slf4j.{Logger, LoggerFactory}


/**
 * User: Maurício Linhares
 * Date: 2/25/12
 * Time: 8:01 PM
 */

object Log {

  def get[T](implicit manifest : Manifest[T] ) = {
    LoggerFactory.getLogger( manifest.erasure.getName )
  }

  def getByName( name : String ) = {
    LoggerFactory.getLogger(name)
  }

}

File: db-async-common/src/main/java/com/github/mauricio/async/db/util/Log.kt
package com.github.mauricio.async.db.util

import org.slf4j.Logger
import org.slf4j.LoggerFactory

object Log {

  fun get(): Logger {
    TODO()
  }

  fun getByName(name: String): Logger {
    return LoggerFactory.getLogger(name)
  }

}
